src/test/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/gitlab/GitlabMergeRequestDecoratorIntegrationTest.java

```diff
@@ -202,11 +203,11 @@ private void decorateQualityGateStatus(QualityGate.Status status) {
                 .willReturn(created()));
 
         wireMockRule.stubFor(post(urlPathEqualTo("/api/v4/projects/" + sourceProjectId + "/merge_requests/" + mergeRequestIid + "/discussions"))
-                .withRequestBody(equalTo("body=summary+comm%C3%A9nt%0A%0A%5Blink+text%5D"))
+                .withRequestBody(equalTo("body=summary+comm%C3%83%C2%A9nt%0A%0A%5Blink+text%5D"))
```

Comment:
I can't see a reason why this line would need to change. This implies that the comment is being double encoded which shouldn't happen as part of your changes.

Reply:
should be fixed with https://github.com/mc1arke/sonarqube-community-branch-plugin/pull/580

Reply:
@mc1arke test issue was solved.